brave-jaxrs2/pom.xml

```diff
@@ -37,6 +37,11 @@
             <version>1</version>
         </dependency>
         <dependency>
+            <groupId>javax.annotation</groupId>
+            <artifactId>javax.annotation-api</artifactId>
+            <version>1.2</version>
```

Comment:
just curious.. this matches servlet 2.5, right? If so, maybe make a comment that it does (so that someone changing below doesn't invalidate this)